cordformation/src/test/resources/postgres.gradle

```diff
@@ -7,8 +7,15 @@ ext {
     dbName                 = 'mydb'
     dbPort                 = 5432
     dbHostName             = 'localhost'
-    dbInit                 = 'Postgres_init.sh'
     dbDockerfile           = 'Postgres_Dockerfile'
+    dbInit                 = 'Postgres_init.sh'
+    dbDataVolume           =  [
+            hostPath      : 'data',
+            containerPath : '/var/lib/postgresql/data:\${PERMISSIONS}',
+            containerPathArgs   : [
+                    PERMISSIONS : "z"
+            ]
```

Comment:
As an aisde, I have _absolutely no idea_ what permissions `z` means?! :exploding_head:

Reply:
z or rw are suffixes used for a volume mount inside a Docker container which tells Docker that the volume that is being attached will be shared. This option needs to be specified because the Postgres docker image, when initialised, changes the folder permissions of the attached volume (folder) for some reason